Gemfile

```diff
@@ -0,0 +1,7 @@
+source "http://www.rubygems.org"
+
+gem 'pry'
```

Comment:
If you'd like to use these we should have them in a development group

Reply:
sounds good